uiowa/uiowa
# [INRC] Grant theming

Resolves https://github.com/uiowa/uiowa/issues/7846

# How to test

```
ddev blt frontend && ddev blt ds --site=inrc.law.uiowa.edu && ddev drush @lawinrc.local uli /grants/iowa-foundation-agricultural-advancement
```
- Code review
- Stress test the layout with maximum and minimum amount of content
- Double check aria is working on application info accordion

There is a code change here that adjusts the work from https://github.com/uiowa/uiowa/pull/7766.  Re-test those instructions and also test the badge/icons on https://housing.uiowa.edu/residence-halls/burge locally and confirm that they are no longer offset.
